deploy/manifests/sample-apps/grpc.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grpcbin
+  labels:
+    app: grpcbin
+spec:
+  ports:
+  - name: grpc
+    port: 80
```

Comment:
The Ingress rule points to 9001 and it works probably because Kong sends traffic directly to the pod and doesn't care about this port. But this should be 9001 as well or the ingress rule should be 80.

Reply:
And apologies for not catching in the earlier reviews. This is the last change we need. Otherwise this is good to go.

Reply:
Done